osdu/.htaccess

```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/Accenture/OSDU-Ontology/blob/main/ttl/OSDU.ttl [R=302,L]
```

Comment:
This is still pointing to the github UI.  You might consider the "raw" link: https://raw.githubusercontent.com/Accenture/OSDU-Ontology/main/ttl/OSDU.ttl.  Or setup your project to publish to github pages and use a link from there.

Reply:
I have changed the link to https://accenture.github.io/OSDU-Ontology/